shared/redux/rtk-apis/assignments/assignments.api.ts

```diff
@@ -58,6 +58,21 @@ const assignmentsApi = projectApi.injectEndpoints({
       }),
       invalidatesTags: ["Assignments"],
     }),
+
+    submitAssignment: builder.mutation<void, { assignmentId: number; userId: number; file: File }>({
+      query: ({ assignmentId, userId, file }) => {
+        const formData = new FormData();
+        formData.append("file", file);
+        formData.append("assignmentId", assignmentId.toString());
+        formData.append("userId", userId.toString());
+        return {
+          url: "/assignments/submit-assignment",
```

Comment:
Please follow REST principles

Should be something like `classrooms/:classroomId/assignments/:assignmentId/subm..`